bindings/rust/build.rs

```diff
@@ -1,64 +1,61 @@
-extern crate bindgen;
-
 use std::env;
 use std::path::PathBuf;
 
 use cmake::Config;
 
 fn main() {
```

Comment:
I like to add `anyhow` as a dev dependency and do `fn main() -> anyhow::Result<()>` so you can use the `?` operator and get better error messages.

Reply:
I'm not 100% convinced that adding a dependency is worth it. If `anyhow` was already used (though I don't think it should be in a library), it was a different case.

Reply:
Okay, doesn't make a huge difference.